### PR TITLE
feat: [PLG-995]:  Added early access and tiles view

### PR DESCRIPTION
**Pull request by devinder-harness**
JIRA Ticket:
https://harness.atlassian.net/browse/PLG-995
### Summary
This PR adds a new tile view to `Resource Center` and Link to `Whats New` and `Early Access` behind FF `EARLY_ACCESS_ENABLED`

#### Screenshots
image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by kapilkumawat86 — APPROVED** [review]

**Comment by bhavya-sinha** [comment]
Early Acess: Spelling is wrong in the screenshot

File: src/modules/10-common/strings/strings.en.yaml [hunk 1]
```diff
@@ -825,10 +825,17 @@ scope: '{{ $.delegate.Scope }}'
 community: Community
 resourceCenter:
   title: 'RESOURCE CENTER'
+  productUpdates:
+    title: Product Updates
+    whatsnew: 'What’s New'
+    earlyAcess: Early Acess
```
[the remark this answers is not in the document]
Comment (devinder-harness) [reply src/modules/10-common/strings/strings.en.yaml]: updated

File: src/modules/10-common/components/ResourceCenter/ResourceCenter.module.scss [hunk 2]
```diff
@@ -104,3 +105,41 @@
...
+      .icon {
+        margin-right: var(--spacing-small);
+      }
+      &:hover {
+        background: rgba(255, 255, 255, 0) !important;
+        border: 1px solid var(--primary-7);
+      }
+      .iconFilled {
+        > svg {
+          fill: #6ad5f8 !important;
```
Comment (nanaxuHarness) [inline src/modules/10-common/components/ResourceCenter/ResourceCenter.module.scss:137]: can we use harness color here

**Review by hekimoglu010 — DISMISSED** [review]

**Review by nanaxuHarness — APPROVED** [review]

**Review by PratyushGar — APPROVED** [review]

**Comment by devinder-harness** [comment]
retrigger sonar

**Review by PratyushGar — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**